README.md

```diff
@@ -62,6 +62,31 @@ Follow linked crate name for detailed status. Please note that all crates follow
 
 [semver]: https://semver.org
 
+### Pipeline Integration
+
+Some CI/CD pipelines leverage repository cloning. Below is a copy-paste-able example to build docker images for such workflows.
+
+#### Pipeline Integration (recommended)
+
+Build an image without a target and then copy the binaries into your local image
+
+```dockerfile
+COPY --from gitoxide:latest /bin/gix /usr/local/bin/
```

Comment:
This works if the `gitoxide` image was already built, right? That wasn't clear to me and I thought maybe it pulls it from the network. Could that be clarified in the paragraph above as well?

Reply:
@Byron Do you have a docker hub account? If so, that should be the "official" image... and `docker` would automatically pull that image and copy the file.

Reply:
Let's leave dockerhub out of the picture for now as I have no plans to maintain an official image for now. This would mean the `gitoxide:latest` name assumes a local image, which probably means the description on how to build that should come first.